examples/whl.py

```diff
@@ -0,0 +1,120 @@
+"""Install from a .whl file.
+"""
+
+import argparse
+import codecs
+import contextlib
+import sys
+import zipfile
+
+from installer._compat import pathlib
+from installer._compat.typing import TYPE_CHECKING
+from installer.layouts import DistInfo
+from installer.records import RecordItem, parse_record_file, write_record_file
+
+if TYPE_CHECKING:
+    from typing import Any, ContextManager, Dict, IO, Iterator
+
+
+class ZipFileInstaller(object):
+    """Install a local wheel.
+    """
+
+    def __init__(self, name, distinfo, zf):
+        # type: (str, DistInfo, zipfile.ZipFile) -> None
+        self._name = name
+        self._distinfo = distinfo
+        self._zf = zf
```

Comment:
can we rename this to zip_file_handler instead, ``zf`` doesn't read well in my mind 🤔